File: 프로그래머스/lv0/120830. 양꼬치/양꼬치.py
def solution(n, k):
    # Calculate the total cost for the skewers
    skewer_cost = n * 12000
    
    # Calculate the number of drinks that were not free
    extra_drinks = max(0, k - n // 10)
    
    # Calculate the total cost for the extra drinks
    drink_cost = extra_drinks * 2000
    
    # Return the total cost
    return skewer_cost + drink_cost

File: 프로그래머스/0/120830. 양꼬치/양꼬치.java
class Solution {
    public int solution(int n, int k) {
        int answer = 0;
        // n*12000+2000*(k-n/10);
        
        answer = n*12000+2000*(k-n/10);
        // 총 얼마를 지불하는지
        return answer;
    }
}